buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -30,12 +29,9 @@
 
 FAIL_IF_MIGRATION_REQUIRED = os.environ.get("USE_BAZELISK_MIGRATE", "").upper() == "FAIL"
 
+FLAG_LINE_PATTERN = re.compile(r"\s*(?P<flag>--\S+)\s*")
 
-INCOMPATIBLE_FLAG_LINE_PATTERN = re.compile(
-    r"\s*(?P<flag>--incompatible_\S+)\s*(\(Bazel (?P<version>.+?): (?P<url>.+?)\))?"
-)
-
-FlagDetails = collections.namedtuple("FlagDetails", ["bazel_version", "issue_url"])
+INCOMPATIBLE_FLAGS = bazelci.fetch_incompatible_flags()  # key: flag name, value: Github URL
```

Comment:
Can we please get rid of this global variable? It'll be very painful to write tests or to run this script locally. It would be better to call the function from inside main()

Reply:
Totally, that's my mistake, fixed!